src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -89,3 +90,63 @@
+
+(rf/reg-event-fx :wallet/send-transaction
+ (fn [{:keys [db]} [sha3-pwd]]
+   (let [route (get-in db [:wallet :ui :send :route])
+         from-address (get-in db [:wallet :current-viewing-account-address])
+         to-address (get-in db [:wallet :ui :send :to-address])
+         from (:From route)
+         token (get-in db [:wallet :ui :send :token])
+         token-id (:symbol token)
+         from-asset token-id
+         to-asset token-id
+         bridge-name (:BridgeName route)
+         chain-id (:chainId from)
+         multi-transaction-command {:fromAddress from-address
+                                    :toAddress   to-address
+                                    :fromAsset   from-asset
+                                    :toAsset     to-asset
+                                    :fromAmount  (:AmountOut route)
+                                    :type        0}
+         transaction-bridge
+         [{:BridgeName bridge-name
+           :ChainID    chain-id
+           :TransferTx {:From                 from-address
+                        :To                   to-address
+                        :Gas                  (money/to-hex (:GasAmount route))
+                        :GasPrice             (money/to-hex (money/->wei :gwei
+                                                                         (:gasPrice (:GasFees route))))
+                        :Value                (:AmountOut route)
+                        :Nonce                nil
+                        :MaxFeePerGas         (money/to-hex
+                                               (money/->wei :gwei
+                                                            (:maxFeePerGasMedium (:GasFees route))))
+                        :MaxPriorityFeePerGas (money/to-hex (money/->wei :gwei
+                                                                         (:maxPriorityFeePerGas
+                                                                          (:GasFees
+                                                                           route))))
+                        :Input                ""
+                        :Data                 "0x"}}]
+         request-params [multi-transaction-command transaction-bridge sha3-pwd]]
+     {:json-rpc/call [{:method     "wallet_createMultiTransaction"
+                       :params     request-params
+                       :on-success (fn [result]
+                                     (rf/dispatch [:hide-bottom-sheet])
+                                     (rf/dispatch [:wallet/add-authorized-transaction result]))
+                       :on-error   (fn [error]
+                                     (log/error "failed to send transaction"
+                                                {:event  :wallet/send-transaction
+                                                 :error  error
+                                                 :params request-params}))}]})))
```

Comment:
This event looks a little large. I think we could split it into more small pieces.

Most of the problems related to why it is verbose are related to CamelCase, so I'd suggest to transform the keys keys while being stored:

```clojure
(rf/reg-event-fx
 :wallet/suggested-routes-success
 (fn [{:keys [db]} [suggested-routes timestamp]]
   (when (= (get-in db [:wallet :ui :send :suggested-routes-call-timestamp]) timestamp)
     {:db (-> db
              (assoc-in [:wallet :ui :send :suggested-routes] suggested-routes)
              ;; Transform the keys in `(first (:Best suggested-routes))` to kebab case
              (assoc-in [:wallet :ui :send :route] (first (:Best suggested-routes)))
              (assoc-in [:wallet :ui :send :loading-suggested-routes?] false))})))
```
That will also solve the problem @OmarBasem said about transforming the estimated time route.

Once that is done, we could refactor this event easier, just as a suggestion:
```clojure
(rf/reg-event-fx
 :wallet/send-transaction
 (fn [{{wallet-db :wallet} :db} [sha3-pwd]]
   (let [{:keys [route token
                 to-address]}      (get-in wallet-db [:ui :send])
         from-address              (:current-viewing-account-address wallet-db)
         token-id                  (:symbol token)
         multi-transaction-command {:fromAddress from-address
                                    :toAddress   to-address
                                    :fromAsset   token-id
                                    :toAsset     token-id
                                    :fromAmount  (:amount-out route)
                                    :type        0}
         transaction-bridge        (get-transaction-bridge {:from-address from-address
                                                            :to-address   to-address
                                                            :route        route})
         request-params            [multi-transaction-command transaction-bridge sha3-pwd]]
     {:json-rpc/call [,,,]})))
```

That is getting the transaction bridge map in a separate function. I'm not sure rn, but I think getting a transaction bridge will be a common operation, so maybe that function can be reused or extended in the future. ATM, it just helps to readability, that function is:
```clojure
(defn- get-transaction-bridge [{:keys [from-address to-address route]}]
  (let [{:keys [from bridge-name amount-out gas-amount gas-fees]}           route
        {:keys [gas-price max-fee-per-gas-medium max-priority-fee-per-gas]} gas-fees]
    [{:BridgeName bridge-name
      :ChainID    (:chain-id from)
      :TransferTx {:From                 from-address
                   :To                   to-address
                   :Gas                  (money/to-hex gas-amount)
                   :GasPrice             (money/to-hex (money/->wei :gwei gas-price))
                   :Value                amount-out
                   :Nonce                nil
                   :MaxFeePerGas         (money/to-hex (money/->wei :gwei max-fee-per-gas-medium))
                   :MaxPriorityFeePerGas (money/to-hex (money/->wei :gwei max-priority-fee-per-gas))
                   :Input                ""
                   :Data                 "0x"}}]))
```

Reply:
Yep, I agree with everything here and about the keys neededing to be adjusted. Will do all! Thanks for the feedback @ulisesmac  & @OmarBasem  :)